config/constants.ts

```diff
@@ -94,6 +94,7 @@ export const getEnvironmentConfig = (
               maxACU: 1,
```

Comment:
Wouldn't mind set this up to 16 ACU as it is serverless ops. It will scale automatically down 0.5 when no load. FM could easily hit 1 ACU limit.